### DISCUSSION TITLE
使用表格识别的时候出现报错： 'MergedCell' object attribute 'value' is read-only

### DISCUSSION THREAD

**Discussion by wzg722** [PaddlePaddle/PaddleOCR#12932] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
 使用表格识别的时候出现报错： 'MergedCell' object attribute 'value' is read-only ，请问官方有解决方案吗？

**Comment by WKangei** [comment]
使用官方的tablepyxl的包，在ppstructure\table\tablepyxl路径下
